pandas/core/arrays/arrow/array.py

```diff
@@ -1269,7 +1269,6 @@ def _mode(self: ArrowExtensionArrayT, dropna: bool = True) -> ArrowExtensionArra
         ----------
         dropna : bool, default True
             Don't consider counts of NA values.
-            Not implemented by pyarrow.
```

Comment:
worth asking pyarrow to implement this upstream?

Reply:
Sure opened https://github.com/apache/arrow/issues/34173